nrel/hive/config/hive_config.py

```diff
@@ -68,12 +68,9 @@ def from_dict(
         root_logger = logging.getLogger("")
         root_logger.setLevel(global_config.log_level)
 
-        if global_config.verbose:
-            log.info(
-                f"global hive configuration loaded from {global_config.global_settings_file_path}"
-            )
-            for k, v in global_config.asdict().items():
-                log.info(f"  {k}: {v}")
+        log.info(f"global hive configuration loaded from {global_config.global_settings_file_path}")
```

Comment:
why remove the effect of the verbose flag?

Reply:
ahh yes, good catch, I was having logging trouble and was using that for trouble shooting, just added back in